### PR TITLE
Added: Button component

### PR DESCRIPTION
**Pull request by pratik-kumar-sahu**
- Added Button component.
- Live link --> https://deploy-preview-4--fuzz-ui.netlify.app/pages/docs.html

### CODE REVIEW

File: styles/_components.css [hunk 1]
```diff
@@ -94,3 +90,71 @@
...
+    min-height: 5rem;
+    background-color: var(--color-primary-dark);
+}
+
+.code-snippet-container {
+    width: 100%;
+    min-height: 10rem;
+    max-height: 25rem;
+    padding: 1rem 1rem 0 1rem;
+    background-color: #2b3e50;
```
Comment (DivyanshSareen) [inline styles/_components.css]: var for colors

File: styles/components/button.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    background-color: #0ea5e9;
+}
+
+.btn-secondary-solid {
+    color: var(--color-white);
+    background-color: var(--color-primary);
+}
+
+.btn-secondary-solid:hover {
+    background-color: #94a3b8;
```
Comment (DivyanshSareen) [inline styles/components/button.css]: var for colors

File: styles/_components.css (hunk 1 above)
Comment (pratik-kumar-sahu) [reply styles/_components.css]: Sure

File: styles/components/button.css (hunk 2 above)
Comment (pratik-kumar-sahu) [reply styles/components/button.css]: Sure

**Review by Mahi24-prog — APPROVED** [review]
Looks good to me pls go ahead

**Pull request merged**